sycl/test-e2e/Matrix/joint_matrix_bf16_fill_k_cache_unroll.cpp

```diff
@@ -8,6 +8,8 @@
 // UNSUPPORTED: target-nvidia, target-amd
 // UNSUPPORTED-INTENDED: aspect-ext_intel_matrix isn't currently supported for
 // other triples
+// XFAIL: run-mode && arch-intel_gpu_bmg_g21
```

Comment:
Does this pass on PVC? only BMG is affected?

Reply:
There are NO PVC run in CI yet.